src/Util/OperationTypes.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ApiPlatform\Core\Util;
+
+final class OperationTypes
```

Comment:
@teohhanhui WDYT about the name? Do you agree we should put this in `ApiPlatform\Core`?